types/knex/index.d.ts

```diff
@@ -13,6 +13,7 @@
 import events = require("events");
 import stream = require ("stream");
 import Bluebird = require("bluebird");
+import { ColumnBuilder } from "knex";
```

Comment:
Your IDE may have auto added this on accident.

Reply:
Very likely.  I'll remove that.